distribution/openrct2.d.ts

```diff
@@ -2874,6 +2874,11 @@ declare global {
          * The total number of frames in the current animation.
          */
         readonly animationLength: number;
+
+        /**
+         * The ride ID of the guest's favourite ride.
+         */
+        favouriteRide: number;
```

Comment:
Can you make it actually return `null` when a guest has no favourite ride? You can look at some other properties that can return both a number and null.
```suggestion
        favouriteRide: number | null;
```